README.md

```diff
@@ -40,18 +40,28 @@ npm run vector_store:upload -- --store=pinecone --repo-path=/path/to/repo --glob
 For example, to search the Phoenix repository's guides, it would look like:
 
 ```
-npm run index:upsert -- --repo-path=/path/to/phoenix --glob-path="guides/**/*.md"
+npm run vector_store:upload -- --store=pinecone --repo-path=/path/to/phoenix --glob-path="guides/**/*.md"
 ```
 
-*Note: Only markdown files are supported right now*
+_Note: Right now, you can only upsert markdown files with index:upsert_
```

Comment:
I changed language to "upload" since not all the vector dbs have upsert. So we probably want to keep language consistent with "upload"